prow/spyglass/gcsartifact_fetcher.go

```diff
@@ -62,15 +68,19 @@ func NewGCSArtifactFetcher(c *storage.Client) *GCSArtifactFetcher {
 	}
 }
 
-func fieldsForJob(src jobSource) logrus.Fields {
+func fieldsForJob(src *gcsJobSource) logrus.Fields {
 	return logrus.Fields{
 		"jobPrefix": src.jobPath(),
 	}
 }
 
 // newGCSJobSource creates a new gcsJobSource from a given bucket and jobPrefix
 func newGCSJobSource(src string) (*gcsJobSource, error) {
-	gcsURL, err := url.Parse(src)
+	split := strings.SplitN(src, "/", 2)
+	if len(split) <= 1 {
+		return &gcsJobSource{}, ErrCannotParseSource
```

Comment:
This error is super opaque -- can we explain why the parsing failed for end user UX?